packages/expo/src/Expo.fx.tsx

```diff
@@ -1,6 +1,7 @@
 // load expo-asset immediately to set a custom `source` transformer in React Native
 import './winter';
 import 'expo-asset';
+import 'expo-system-ui';
```

Comment:
This module is not marked as a side-effect and will break with tree shaking https://docs.expo.dev/guides/tree-shaking/#side-effects
Also shouldn't pull in all of expo-system-ui to install a global on the runtime. This assumes all users need `expo-system-ui`.

Reply:
> Also shouldn't pull in all of expo-system-ui to install a global on the runtime. This assumes all users need expo-system-ui.

I've been told that `expo-constants` and `expo-system-ui` (needed for `userInterfaceStyle` and `backgroundColor`) work to merge it into the core was started, so I though it was not an issue.